src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/integration/AttendancesIntegrationTest.kt

```diff
@@ -152,17 +151,17 @@ class AttendancesIntegrationTest : IntegrationTest() {
                 .bookingId(2)
                 .build()))
 
-    val response =
-        restTemplate.exchange(
-            "/attendances/LEI?date={0}&period={1}&bookings=${1}&bookings=${2}",
-            HttpMethod.GET,
-            createHeaderEntity(""),
-            AttendancesResponse::class.java,
-            LocalDate.of(2019, 10, 10),
-            TimePeriod.PM)
-
-    assertThat(response.statusCodeValue).isEqualTo(200)
-    assertThat(response.body?.attendances).extracting("bookingId").contains(1L, 2L)
+    webTestClient
+      .get()
+      .uri("/attendances/LEI?date={0}&period={1}&bookings=${1}&bookings=${2}", LocalDate.of(2019, 10, 10), TimePeriod.PM)
```

Comment:
Is it worth using uri builder pattern with queryParam here?

Reply:
When I was looking at the docs, they sort of implied that you can use the builder if you "need full programmatic control". That seemed a bit excessive, so I moved back to this. But happy to change.

Reply:
Happy either way but let's pick one and make everything else consistent.  I think it would look cleaner to use the builder here because you could just use:
`queryParam("bookings', setof(1,2,3,4,))`

Reply:
Updated to uri builder everywhere :)